test/lib/instruction-parser.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { getPackagesFromRunInstructions } from '../../lib/instruction-parser';
+
+describe('instruction parsers', () => {
+  describe('getPackagesFromRunInstructions', () => {
+    test('supports "apt install"', () => {
+      const instruction = 'RUN apt install curl';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({ 'curl': { instruction } });
+    });
+
+    test('supports "apt-get install"', () => {
+      const instruction = 'RUN apt-get install curl';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({ 'curl': { instruction } });
+    });
+
+    test('supports "apk add"', () => {
+      const instruction = 'RUN apk add curl';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({ 'curl': { instruction } });
+    });
+
+    test('supports "rpm -i"', () => {
+      const instruction = 'RUN rpm -i curl';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({ 'curl': { instruction } });
+    });
+
+    test('supports "rpm --install"', () => {
+      const instruction = 'RUN rpm --install curl';
+      const packages = getPackagesFromRunInstructions([instruction]);
+
+      expect(packages).toEqual({ 'curl': { instruction } });
+    });
+
+    test('ignores irrelevant flags', () => {
```

Comment:
can we add a test where the flags are in the end?
`apt-get install vim -y`

Reply:
Good catch